src/WebView.android.tsx

```diff
@@ -10,6 +10,8 @@ import {
   findNodeHandle,
 } from 'react-native';
 
+import BatchedBridge from 'react-native/Libraries/BatchedBridge/BatchedBridge';
```

Comment:
This would make webview dependent on a specific version of React Native. Peer dependencies need to be updated & docs to make sure users know this fix can only support a certain RN version

Reply:
I'm not sure I follow. BatchedBridge has been there since RNv0.6.0, no?

Reply:
It's probably fine, but if React Native changes where the BatchedBridge lives, this could break. I'm not too worried about it. Try changing it to `/foo` and see how it breaks, and perhaps we could check for that and provide a nicer error message.

Reply:
Thank you, I will give it a try and see if I can add an error message.